day10/C++/day10AllPermutations.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+* @author : YashMeh
+* @date   : 28/1/2019
+*/
+#include<bits/stdc++.h>
+using namespace std;
+//Using STL
+void permute(string s)
+{
+   sort(s.begin(),s.end());
```

Comment:
Tested without this line of code and the algo still works. This is because according to the [docs](https://en.cppreference.com/w/cpp/algorithm/next_permutation), next_permutation
> Returns true if such permutation exists, otherwise transforms the range into the first permutation (as if by std::sort(first, last)) and returns false